Contributing.md

```diff
@@ -0,0 +1,27 @@
+# Contributing Guidelines
+
+This is a open source project and we love to receive contributions from community --- you!. This project is currently in alpha state.
+
+## Contributing code and documentation changes
+
+### **DISCLAIMER**
+
+- Please ask for clarification in issues/ pull requests to get more understanding on tasks that you are doing.
+- Don't force push your branch, when changes are required / until you are asked to for latest changes.
+
+You will need to fork the [Main Repository](https://github.com/nerds-amp-mods/Hackathon-Login-Starter-Pack.git) to your github account and clone the repository to your local machine.
+
+Further Information for specific projects given below
+
+## Backend
+
+- Currently backend supports python 3.9
```

Comment:
If we're developing with 3.9 we don't need to import `typing`

Reply:
Umm. Good call

Reply:
Hmm we support from 3.8.5 . Gotta change that